advanced_alchemy/extensions/litestar/plugins/init/config/asyncio.py

```diff
@@ -39,29 +39,47 @@
 )
 
 
-async def default_before_send_handler(message: Message, scope: Scope) -> None:
-    """Handle closing and cleaning up sessions before sending.
-
+def default_handler_maker(
+    session_scope_key: str = SESSION_SCOPE_KEY,
```

Comment:
Does this need the default value? Is there some use case where we call this and don't have access to the config?

Reply:
Nope, it doesn't, but if I move it to the end where it should be, it'll need it.

The main reason this is there is so that we can create the `default_before_send_handler` just below this declaration.